gapic/samplegen/samplegen.py

```diff
@@ -944,17 +953,19 @@ def generate_sample_specs(api_schema: api.API, *, opts) -> Generator[Dict[str, A
 
     for service_name, service in gapic_metadata.services.items():
         api_short_name = api_schema.services[f"{api_schema.naming.proto_package}.{service_name}"].shortname
-        for transport_type, client in service.clients.items():
-            if transport_type == "grpc-async":
-                # TODO(busunkim): Enable generation of async samples
-                continue
+        for transport_name, client in service.clients.items():
+            if transport_name == "grpc-async":
+                transport_type = "async"
+            else:
+                transport_type = "sync"
             for rpc_name, method_list in client.rpcs.items():
                 # Region Tag Format:
                 # [{START|END} ${apishortname}_generated_${api}_${apiVersion}_${serviceName}_${rpcName}_{sync|async}_${overloadDisambiguation}]
                 region_tag = f"{api_short_name}_generated_{api_schema.naming.versioned_module_name}_{service_name}_{rpc_name}_{transport_type}"
                 spec = {
                     "sample_type": "standalone",
                     "rpc": rpc_name,
+                    "transport": transport_name,
```

Comment:
Should this be `transport_type`?  I think `transport_name` would raise `NameError` here.

Reply:
`transport_name` is created on 956 and is one of `grpc`, `grpc-async` or `rest`